lib/homepage.dart

```diff
@@ -45,6 +45,8 @@ class _MyHomePageState extends State<MyHomePage> {
   bool highlightsLoaded = false;
   bool sliderPostersLoaded = false;
   bool announcementsLoaded = false;
+  DateTime YESTERDAY_DATE = new DateTime(
```

Comment:
An issue might occur if the current day is the 1st of the month. Maybe you can try:

`DateTime.now().subtract(Duration(days:1))`

Reply:
good catch, thanks!!